src/functions.jl

```diff
@@ -343,6 +343,11 @@ for (Ts, Ss) in [(Rlist, Rlist), (Rlist, Olist), (Olist, Rlist)]
 
     end
 end
+
+# Specific methods to resolve ambiguities in 1.6
+Base.:*(x::Adjoint{<:Any, <:AbstractMatrix{T}}, y::KeyedVector{S}) where {T, S<:Number} = matmul(x,y)
+Base.:*(x::Transpose{<:Any, <:AbstractMatrix{T}}, y::KeyedVector{S}) where {T, S<:Number} = matmul(x, unname(y))
```

Comment:
> * is not defined on Transpose and NamedDimsArray.

Doesn't that need to be fixed in NamedDims.jl?
I thought we had that one

Reply:
Yeah, I wonder why it doesn't hit this?

https://github.com/invenia/NamedDims.jl/blob/master/src/functions_math.jl#L53

Reply:
> Yeah, I wonder why it doesn't hit this?
> 
> https://github.com/invenia/NamedDims.jl/blob/master/src/functions_math.jl#L53

I think it's because that method takes `Transpose{<:Real,<:AbstractVector{<:Real}}` as opposed to `Transpose{<:Real,<:AbstractMatrix{<:Real}}`.

I think I can patch that in NamedDims though by adding those types.